chat-plugins/tcg.js

```diff
@@ -0,0 +1,34 @@
+/**
+* TCG: Yugioh wiki plugin
+* This is a command that allows users to search the yugioh wiki for cards. It will display the closest match with a given query, or a separate message if there isn't anything found.
+*/
+
+exports.commands = {
+	ygo: 'yugioh',
+	yugioh: function (target, room, user) {
+		if (room.id !== 'tcg') return this.sendReply("This command can only be used in the TCG room.");
+		if (!this.canBroadcast()) return;
+
+		// The result is fetched from the wikia api with an XMLHttpRequest.
```

Comment:
Why the external dependency?

Doesn't Node have built-in support for GET requests? Just use that. -_-